types/jest/index.d.ts

```diff
@@ -131,6 +131,26 @@ declare namespace jest {
      * Returns the number of fake timers still left to run.
      */
     function getTimerCount(): number;
+    /**
+     * Set the current system time used by fake timers. Simulates a user
+     * changing the system clock while your program is running. It affects the
+     * current time but it does not in itself cause e.g. timers to fire; they
+     * will fire exactly as they would have done without the call to
+     * jest.setSystemTime().
+     *
+     * > Note: This function is only available when using modern fake timers
+     * > implementation
+     */
+    function setSystemTime(now?: number | Date): void;
```

Comment:
I tried to follow the patterns I observed elsewhere in this file here, I just copy pasted these comments straight out of jest's docs.